index.html

```diff
@@ -244,9 +269,71 @@
         </dt>
       </dl>
       <p>
-        The <code>PermissionName</code> enum defines the list of known
-        permission names.
+        Each enumerator-value in the <a>PermissionName</a> enum identifies a
+        <dfn data-export="">permission</dfn>, which consists of the following
+        algorithms and types:
       </p>
+      <dl>
+        <dt>
+          An <dfn data-export="">associated PermissionDescriptor</dfn>
+        </dt>
+        <dd>
+          A subtype of <a>PermissionDescriptor</a>.
```

Comment:
Looking at all the examples, it seems like it's often not a subtype. Not sure how best to phrase it; maybe just a parenthatical (or PermissionDescriptor itself if a custom type is not needed).